server/utils/mcp.ts

```
@@ -5,6 +5,62 @@ import { queryCollection } from '@nuxt/content/server'
+/**
+ * Maps legacy documentation paths to current content paths.
+ *
+ * nuxt.com route rules redirect old URLs in the browser, but MCP resolves
+ * content by path via `queryCollection` and does not follow HTTP redirects.
+ */
+export function resolveDocumentationPath(path: string): string {
+  if (path === '/docs/4.x/guide/going-further/error-handling' || path === '/docs/5.x/guide/going-further/error-handling') {
+    return path.replace('/guide/going-further/error-handling', '/getting-started/error-handling')
+  }
+
+  let resolved = path.replace(
```

ESLint

[error] 'resolved' is never reassigned. Use 'const' instead.

(prefer-const)